src/tpg_test_mgmt_cli.c

```diff
@@ -105,19 +105,19 @@ struct cmd_tests_start_stop_result {
 };
 
 static cmdline_parse_token_string_t cmd_tests_start_T_start =
-    TOKEN_STRING_INITIALIZER(struct cmd_tests_start_stop_result, start, "start");
+TOKEN_STRING_INITIALIZER(struct cmd_tests_start_stop_result, start, "start");
```

Comment:
Please follow the initial alignment and indentation style of the file. This applies to the whole pull request.

The main guidelines are:
- use spaces not tabs.
- the continuation of a statement on a new line should be indented with 4 spaces.
- function arguments both for function definitions and function calls should be aligned with the open parenthesis `(`.
- align variable definitions that are in the same block so the variable names (including `*`) are aligned:

    ```
    int    a;
    float *p;
    long   b;
    
    long unsigned x;
    double        y;
    ```